Tests/Unit/Twig/PagerExtensionTest.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+/*
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ONGR\FilterManagerBundle\Tests\Unit\Twig;
+
+use ONGR\FilterManagerBundle\Twig\PagerExtension;
+use Symfony\Component\Routing\RouterInterface;
+
+class PagerExtensionTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * Returns instance of pager extension.
+     *
+     * @param bool $includeGenerate
+     *
+     * @return PagerExtension
+     */
+    protected function getPager($includeGenerate = true)
+    {
+        /** @var $router RouterInterface|\PHPUnit_Framework_MockObject_MockObject $environment */
```

Comment:
should be `@var RouterInterface|\PHPUnit_Framework_MockObject_MockObject $router`.